src/Build/BackEnd/Shared/TargetResult.cs

```diff
@@ -136,6 +136,21 @@ public TargetResultCode ResultCode
             }
         }
 
+        public string TargetResultCodeToString()
+        {
+            switch (ResultCode)
+            {
+                case TargetResultCode.Failure:
+                    return nameof(TargetResultCode.Failure);
+                case TargetResultCode.Skipped:
+                    return nameof(TargetResultCode.Skipped);
+                case TargetResultCode.Success:
+                    return nameof(TargetResultCode.Success);
+                default:
+                    return "";
```

Comment:
Is there no possibility ResultCode could be something else? The real Enum.ToString() has additional logic to handle numerical values that don't map to named values.